operator-common/src/main/java/io/strimzi/operator/common/AbstractOperator.java

```diff
@@ -259,15 +303,25 @@ public void accept(KubernetesClientException e) {
      */
     private void handleResult(Reconciliation reconciliation, AsyncResult<Void> result) {
         if (result.succeeded()) {
+            successfulReconciliationsCounter.increment();
             log.info("{}: reconciled", reconciliation);
         } else {
             Throwable cause = result.cause();
             if (cause instanceof InvalidConfigParameterException) {
+                failedReconciliationsCounter.increment();
                 log.warn("{}: Failed to reconcile {}", reconciliation, cause.getMessage());
             } else if (!(cause instanceof UnableToAcquireLockException)) {
+                failedReconciliationsCounter.increment();
```

Comment:
isn't it enough one `failedReconciliationsCounter.increment();` call out of this `if` and `elseif` statements?

Reply:
No, because I do not think we should report when the reconciliation fails to obtain a lock as a failure.